client/dashboard/utils/site-url.ts

```diff
@@ -10,3 +11,26 @@ import type { Site } from '../data/types';
 export function getSiteDisplayUrl( site: Site ) {
 	return site.URL.replace( 'https://', '' ).replace( 'http://', '' );
 }
+
+/**
+ * Returns the URL for editing the site.
+ */
+export function getSiteEditUrl( site: Site, isFSEActive?: boolean ) {
```

Comment:
The original function name should be getSiteEditorUrl but I changed it to get getSiteEditUrl since it might also return customizer URL.